docs/advanced-guide/handling-file/page.md

```diff
@@ -57,6 +57,41 @@ func main() {
 }
 ```
 
+### AWS S3 Bucket as File-Store
+
+To run S3 File-Store locally we can use localstack,
+``docker run --rm -it -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack``
+
+
+```go
+package main
+
+import (
+    "gofr.dev/pkg/gofr"
+
+    "gofr.dev/pkg/gofr/datasource/file/s3"
+)
+
+func main() {
+    app := gofr.New()
+	
+	// Note that currently we do not handle connections through session token.
+	// BaseEndpoint is not necessary while connecting to AWS as it automatically resolves it on the basis of region.
+	// However, in case we are using any other AWS compatible service, such like running or testing locally, then this needs to be set.
+	// Note that locally, AccessKeyID & SecretAccessKey is not checked if we use localstack.
+	app.AddFileStore(s3.New(&s3.Config{
+		EndPoint:        "http://localhost:4566",
+		BucketName:      "gofr-bucket-2",
+		Region:          "us-east-1",
+		AccessKeyID:     "test",
+		SecretAccessKey: "test",
```

Comment:
@coolwednesday - is there a way to provide accesskey and secret from environment? Putting it in code is not a good practice.